lib/bob.ex

```diff
@@ -5,7 +5,7 @@ defmodule Bob do
 
     cond do
       silent?(input) -> "Fine. Be that way!"
-      shouting_question?(input) -> "Calm down, I know what I'm doing!"
+      shouting?(input) && question?(input) -> "Calm down, I know what I'm doing!"
```

Comment:
I am an annoying pedant and I think the function calls need to happen in alphabetical order.
```suggestion
      question?(input) && shouting?(input) -> "Calm down, I know what I'm doing!"
```